packages/documentation/src/stories/internet-header/components/internet-header.stories.tsx

```diff
@@ -64,12 +65,12 @@ function mockPage(story: any) {
     <div class="page-wrapper">
       ${story()}
       <main class="container mt-huge-r">
-        <swisspost-internet-breadcrumbs/>
-        <h1 class="mt-huge-r mb-big-r bold">Swiss Post Internet Header</h1>
+        <swisspost-internet-breadcrumbs />
+        <h1 class="mt-huge-r mb-big-r bold">Design System Internet Header</h1>
```

Comment:
This makes a test fail

Reply:
Can't we change the test instead?
If not, just keep it as is.

Reply:
Yes you're totally right